### PR TITLE
Feature/201/create preview for typography collection

### PR DESCRIPTION
**Pull request by ruaraikirk**
Created preview in CMS for text styles (typography) collection. Minimal styling applied just to show concept as design input required.

### CODE REVIEW

File: src/components/Typography/Typography.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    return (
+    <section>
+        <table>
+            <tbody>
+                <tr>
+                <td>
+                    <div
+                        css={css`
+                            text-transform: ${textCase};
+                            font-family: ${style72};
```
Comment (devinea) [inline src/components/Typography/Typography.js]: font-weight: `${style}`

**Comment by ruaraikirk** [comment]
Latest commit with minor updates. Investigation still required on 72 Typeface...

cms.js
```
require('../css/72.css')
```

typography.js
```
require('../../css/72.css')
```


Neither worked, needs further investigation.

**Comment by ruaraikirk** [comment]
Updated and ready for review.

File: src/components/Typography/Typography.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { css } from '@emotion/core'
+
+class Typography extends React.Component {
+  render() {
+
+    const { name, uppercase, style, color, size, height } = this.props;
+    const textCase = uppercase === true ? "uppercase" : "lowercase";
```
Comment (ckoutsiaris) [inline src/components/Typography/Typography.js]: Probably you don't need the ``` === true``` part.

Comment (ruaraikirk) [reply src/components/Typography/Typography.js]: Thanks. Updated and checked.

**Pull request merged**